src/utils/sourcemaps.rs

```diff
@@ -635,8 +669,71 @@ impl SourceMapProcessor {
                 continue;
             }
 
-            if let Some(Some(sourcemap_url)) = self.sourcemap_references.get(&source.url) {
-                source.set_sourcemap_reference(sourcemap_url.to_string());
+            if let Some(Some(sourcemap)) = self.sourcemap_references.get(&source.url) {
+                source.set_sourcemap_reference(sourcemap.url.to_string());
+            }
+        }
+        Ok(())
+    }
+
+    /// Adds debug id reference from the given source map to all minified files.
```

Comment:
What is meant by "the given source map" here?

Reply:
The referenced source map. I changed the text to be more clear.